package.json

```diff
@@ -107,6 +109,7 @@
 		}
 	},
 	"nyc": {
+		"extends": "@istanbuljs/nyc-config-typescript",
```

Comment:
Why this change?

Reply:
This was to support Typescript for the code coverage, but it works even without, just by enabling the source map generation.